### ISSUE TITLE
[FEATURE REQUEST] OAuth Action Missing Parameters ??

### ISSUE THREAD

**Issue by kaiAsmOne** [citrix/terraform-provider-citrixadc#1046]
# Contact us

> For any immediate issues or help , reach out to us at NetScaler-AutomationToolkit@cloud.com !

Hi, 

This one is urgent for us to be able to migrate away from SAML2 to OAuth 2.0 V2.0 API to use an MSAL Enabled Auth endpoint in Azure (not ADAL)

When creating an OAuth Action using citrixadc_authenticationoauthaction i cannot find a way to set the following (arguments missing)

1: ID Token Decrypt Endpoint
2: User Info URL
3: Cert Endpoint
4: Skew time
5: Issuer
6: Grant Type

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by rohit-myali** [comment]
Hi @kaiAsmOne 

I did check the resource `citrixadc_authenticationoauthaction`. I see that we support all the above-mentioned attributes.  We have released this in v1.12.0 (March 15, 2022). 
link to the documentation: https://registry.terraform.io/providers/citrix/citrixadc/latest/docs/resources/authenticationoauthaction

Sample terraform script for the same:
```hcl

resource "citrixadc_authenticationoauthaction" "tf_authenticationoauthaction" {
  name                   = "tf_authenticationoauthaction"
  authorizationendpoint  = "https://example.com/"
  tokenendpoint          = "https://ssexample.com/"
  clientid               = "cliId"
  clientsecret           = "secret"
  resourceuri            = "http://www.sd.com"
  skewtime               = 10
  userinfourl            = "https://example.com/"
  idtokendecryptendpoint = "https://example.com/"
  certendpoint           = "https://example.com/"
  granttype              = "PASSWORD"
  issuer                 = "1.1.1.1"
}
```

Am I missing something here @kaiAsmOne ?

**Comment by kaiAsmOne** [comment]
Hi, 

I dont understand what i have been reading because i could not find this at 

https://registry.terraform.io/providers/citrix/citrixadc/latest/docs/resources/authenticationoauthaction

But it is all there. 

Im so happy to see all attributes i need. Im sorry for wasting your time.
